app/screens/home/search/search.tsx

```diff
@@ -75,7 +75,7 @@ const SearchScreen = ({teamId}: Props) => {
         ]);
 
         setPostIds(postResults?.order?.length ? postResults.order : emptyPostResults);
-        setFileInfos(fileResults?.file_infos?.length ? Object.values(fileResults.file_infos) : emptyFileResults);
+        setFileInfos(fileResults?.file_infos && Object.keys(fileResults?.file_infos) ? Object.values(fileResults.file_infos) : emptyFileResults);
```

Comment:
@larkox, thanks for the work on this while I was out! 

I made this change so that the fileInofs are extracted correctly. They are an object.

Reply:
Checking for Object.keys does nothing, as far as I understand. The real solution here is:
```suggestion
        const fileInfos = fileResults?.file_infos && Object.values(fileResults.file_infos);
        setFileInfos(fileInfos?.length ? Object.values(fileResults.file_infos) : emptyFileResults);
```